### DISCUSSION TITLE
Uninstallation of v

### DISCUSSION THREAD

**Discussion by jpx40** [vlang/v#17830] [Questions and Answers]
Hey, how can I uninstall V on macOS?

**Comment by JalonSolov** [comment]
How did you install it?

If you cloned it and built from source, then just delete the directory, and delete the `/usr/local/bin/v` link.

If you installed it via brew or something, brew probably has an `uninstall` option.

**Reply by jpx40** [reply]
Thank you, you helped me, it was built from source.
